src/webview/WebViewScreen.tsx

```diff
@@ -160,6 +197,14 @@ function WebViewScreen({ route, navigation }: Props) {
           handleSetNavigationTitle(navState.url, navState.title, navState.loading)
         }}
       />
+      {Platform.OS === 'android' && (
+        <WebViewAndroidBottomSheet
+          url={currentUrl}
+          isVisible={showingBottomSheet}
+          onClose={() => toggleBottomSheet()}
```

Comment:
`() => setShowingBottomSheet(false)` here?

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2565/commits/8a36cf8a3596e13c40534c2b37bf18ad499d2f9d!